### PR TITLE
Modify the Readwrite-Splitting document in RQL Syntax

### PR DESCRIPTION
**Pull request by yx9o**
Modify the Readwrite-Splitting document in RQL Syntax.

### CODE REVIEW

File: docs/document/content/features/dist-sql/syntax/rql/rql-readwrite-splitting-rule.en.md [hunk 1]
```diff
@@ -11,22 +11,22 @@ SHOW READWRITE_SPLITTING RULES [FROM schemaName]
...
+| name                        | Rule name                            |
+| auto_aware_data_source_name | Auto-Aware discovery data source name（Display configuration dynamic read-write separation rules）|
+| write_data_source_name      | Write data source name                |
+| read_data_source_names      | Read data source name list            |
+| load_balancer_type          | Load balance algorithm type           |
+| load_balancer_props         | Load balance algorithm parameter      |
 
 ## Example
 
 *Static Read-write Separation Rules*
```
Comment (RaigorJiang) [inline docs/document/content/features/dist-sql/syntax/rql/rql-readwrite-splitting-rule.en.md:25]: Thank you for your PR, can you change these ‘Read-write Separation’ to ‘Readwrite splitting’ by the way?

Comment (yx9o) [reply docs/document/content/features/dist-sql/syntax/rql/rql-readwrite-splitting-rule.en.md]: > Thank you for your PR, can you change these ‘Read-write Separation’ to ‘Readwrite splitting’ by the way?

Done, please review, thank you.

**Review by RaigorJiang — APPROVED** [review]

File: docs/document/content/features/dist-sql/syntax/rql/rql-readwrite-splitting-rule.en.md (hunk 1 above)
Comment (RaigorJiang) [reply docs/document/content/features/dist-sql/syntax/rql/rql-readwrite-splitting-rule.en.md]: 👍  It's OK now. Thank you!

**Comment by yx9o** [comment]
@tristaZero Hi, please review, thank you.

**Review by tristaZero — APPROVED** [review]

**Pull request merged**